docker-images/kafka/kafka-thirdparty-libs/2.3.x/pom.xml

```diff
@@ -17,7 +17,7 @@
 
     <properties>
         <strimzi-oauth.version>0.4.0</strimzi-oauth.version>
-        <cruise-control.version>2.0.100</cruise-control.version>
+        <cruise-control.version>2.0.101</cruise-control.version>
```

Comment:
2.3 is not supported anymore - so this should be removed. I opened #2935 for it.